sdk/containerregistry/azure-containerregistry/azure/containerregistry/_anonymous_exchange_client.py

```diff
@@ -44,9 +45,8 @@ def __init__(  # pylint: disable=missing-client-constructor-parameter-credential
             **kwargs
         )
 
-    def get_acr_access_token(  # pylint:disable=client-method-missing-tracing-decorator
-        self, challenge: str, **kwargs
-    ) -> Optional[str]:
+    @distributed_trace
+    def get_acr_access_token(self, challenge: str, **kwargs) -> Optional[str]:
```

Comment:
Given the method name "get access token", do we really want to trace it?

Reply:
After re-thinking, it's more appropriate to not trace. I'll change them back.